back up tic

```diff
@@ -0,0 +1,562 @@
+			round++;
+			buildArray(idTwo);
+		} else if (board[1][1] === 0){
+			idFive.classList.add("yellow");
+			idFive.innerHTML = "O";
+			round++;
+			buildArray(idFive);
+		} else if (board[2][1] === 0){
+			idEight.classList.add("yellow");
+			idEight.innerHTML = "O";
+			round++;
+			buildArray(idEight);
+		}
+	}else if ((board[0][2] + board[1][2] + board[2][2]) === 10){
+		if (board[0][2] === 0){
+			idThree.classList.add("yellow");
+			idThree.innerHTML = "O";
+			round++;
+			buildArray(idThree);
+		} else if (board[1][2] === 0){
+			idSix.classList.add("yellow");
+			idSix.innerHTML = "O";
+			round++;
+			buildArray(idSix);
+		} else if (board[2][2] === 0){
+			idNine.classList.add("yellow");
+			idNine.innerHTML = "O";
+			round++;
+			buildArray(idNine);
+		}
+	//diagonal
+	}else if ((board[0][0] + board[1][1] + board[2][2]) === 10){
+		if (board[0][0] === 0){
+			idOne.classList.add("yellow");
+			idOne.innerHTML = "O";
+			round++;
+			buildArray(idOne);
+		} else if (board[1][1] === 0){
+			idTwo.classList.add("yellow");
+			idTwo.innerHTML = "O";
+			round++;
+			buildArray(idTwo);
+		} else if (board[2][2] === 0){
+			idNine.classList.add("yellow");
+			idNine.innerHTML = "O";
+			round++;
+			buildArray(idNine);
+		}
+	}else if ((board[0][2] + board[1][1] + board[2][0]) === 10){
+		if (board[0][2] === 0){
+			idThree.classList.add("yellow");
+			idThree.innerHTML = "O";
+			round++;
+			buildArray(idThree);
+		} else if (board[1][1] === 0){
+			idTwo.classList.add("yellow");
+			idTwo.innerHTML = "O";
+			round++;
+			buildArray(idTwo);
+		} else if (board[2][0] === 0){
+			idSeven.classList.add("yellow");
+			idSeven.innerHTML = "O";
+			round++;
+			buildArray(idSeven);
+		}
+	} else {
+		return false;
+	}
+};
+
+var compMove = function (){
+	if (board[1][1] === 0){
+		idFive.classList.add("yellow");
+		idFive.innerHTML = "O";
+		round++;
+		buildArray(idFive);
+	} else if (board[0][2] === 0){
+		idThree.classList.add("yellow");
+		idThree.innerHTML = "O";
+		round++;
+		buildArray(idThree);
+	}else if (board[2][2] === 0){
+		idNine.classList.add("yellow");
+		idNine.innerHTML = "O";
+		round++;
+		buildArray(idNine);
+	}else if (board[2][0] === 0){
+		idSeven.classList.add("yellow");
+		idSeven.innerHTML = "O";
+		round++;
+		buildArray(idSeven);
+	} else if (board[1][0] === 0){
+		idFour.classList.add("yellow");
+		idFour.innerHTML = "O";
+		round++;
+		buildArray(idFour);
+	} else if (board[0][0] === 0){
+		idOne.classList.add("yellow");
+		idOne.innerHTML = "O";
+		round++;
+		buildArray(idOne);
+	}else if (board[1][2] === 0){
+		idSix.classList.add("yellow");
+		idSix.innerHTML = "O";
+		round++;
+		buildArray(idSix);
+	}else if (board[2][1] === 0){
+		idEight.classList.add("yellow");
+		idEight.innerHTML = "O";
+		round++;
+		buildArray(idEight);
+	}else if (board[0][1] === 0){
+		idTwo.classList.add("yellow");
+		idTwo.innerHTML = "O";
+		round++;
+		buildArray(idTwo);
+	}
+
+};
+
+var clickedBox = function (element){
```

Comment:
This function, as well as a couple others, have already been defined. It may have been an error when copying/pasting, but it's good to keep in mind that variables should only be declared once.